node/clients/admin.ts

```diff
@@ -17,12 +17,12 @@ export class Admin extends ExternalClient {
     )
   }
 
-  public exportCatalogToEmail = (email: string, locale: string) =>
+  public exportCatalogToEmail = (email: string) =>
     this.http.post(`StockKeepingUnitReport/ExportToEmail/`, {
       Email: email,
       Filters: {
         StoreId: 0,
       },
-      Locale: locale,
+      Locale: 'en-US',
```

Comment:
Is this temporary while the PR is in draft?

Reply:
Nope.
This locale only changes the headers of the CSV columns, it does not affect data.
We use the name of the headers to recognise the translatable fields on the front end, so allow them to vary by locale would only add more complexity to the front end logic.